00-boilerplate/webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+const basePath = __dirname;
+
+module.exports = {
+  context: path.join(basePath, "src"),
+  resolve: {
+    extensions: [".js", ".ts", ".tsx"],
+  },
+  entry: {
+    app: ["./index.tsx", "./styles.css"],
+  },
+  devtool: "eval-source-map",
+  stats: "errors-only",
+  output: {
+    filename: "[name].[chunkhash].js",
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        exclude: /node_modules/,
+        loader: "babel-loader",
+      },
+      {
+        test: /\.(png|jpg)$/,
+        exclude: /node_modules/,
+        loader: "url-loader",
+      },
```

Comment:
you should use the new asset loader from webpack, will include an example

Reply:
done